src/_locales/en/messages.json

```
@@ -507,6 +507,54 @@
         "message": "Open options page",
         "description": "Button label"
     },
+    "disallowedHeader": {
+        "message": "This song is not being scrobbled",
+        "description": "The header of 'disallowed' popup"
+    },
+    "disallowedDesc1": {
+        "message": "Web Scrobbler has detected the song $2 by $1, but will not scrobble it.",
```

```suggestion
        "message": "Web Scrobbler has detected the song \"$2\" by \"$1\", but will not scrobble it.",
```

I think we could quote the artist and title or perhaps make it bold to make it standout more what was matched? What do you think?

---

I agree. Since we have arbitrary input here I think we should stay out of any parsing and just use quotes. Sounds like more trouble than it’s worth.